src/main/java/org/spongepowered/common/command/WrapperICommandSender.java

```diff
@@ -119,7 +120,7 @@ public MinecraftServer getServer() {
         if (world != null) {
             return world.getMinecraftServer();
         }
-        return null;
+        return SpongeImpl.getServer();
```

Comment:
In which case a command shouldn't be run anyway, and doing so will result in an `InvalidStateException` - perfectly reasonable for that circumstance.